action/action_plugins/include/action_plugins/acceleration_action.h

```diff
@@ -4,7 +4,7 @@
 #include <cmath>
 #include <limits>
 
-#include <pluginlib/class_list_macros.h>
+#include <pluginlib/class_list_macros.hpp>
```

Comment:
Since I can't comment on unchanged lines, I will comment here.

First, good catch about using `.hpp` instead of `.h`.
It looks like .h is deprecated in ROS2 (https://github.com/ros/pluginlib/blob/90c4fb4684f1d6e85f458e8f00cb747383d47be5/pluginlib/include/pluginlib/class_list_macros.h#L37)

However, there are files that are still using `<pluginlib/class_list_macros.h>` instead of <pluginlib/class_list_macros.hpp>`. Please check `follow_route_action.cpp` and `lane_change_action.cpp`. They were probably overlooked since there are at the bottom of the file.

Reply:
Fixed.